features/ACM/upgrade_risks.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Upgrade risks
+
+
+  Scenario: Displaying 1 cluster with high upgrade risk
+    Given user USER1 owns a managed cluster
+      And an upgrade risk has been detected for this cluster
+      And the user USER1 is already logged in to Advanced Cluster Management
+     When user USER1 looks at Advanced Cluster Management / Clusters / Cluster page
+     Then the "Upgrade risk" section displays
+          | Upgrade risk  | High  |
+      And the "Upgrade risk" section is clickable
+     When user USER1 clicks on the "Upgrade risk" section
+     Then the browser opens the details page on console.redhat.com (Advisor / Cluster / Upgrade risk)
+
+
+  Scenario: Displaying 1 cluster with low upgrade risk
+    Given user USER1 is part of account (organization) ACCOUNT1
+      And account (organization) ACCOUNT1 owns a managed cluster
+      And no or low upgrade risk has been detected for this cluster
+      And the user USER1 is already logged in to Advanced Cluster Management
+     When user USER1 looks at Advanced Cluster Management / Clusters / Cluster page
+     Then the "Upgrade risk" section displays
+          | Upgrade risk  | Low |
+      And the "Upgrade risk" section is not clickable
+
+
+  Scenario: Displaying the overview of the clusters at risk
+    Given user USER1 owns one or more managed clusters
+      And the user USER1 is already logged in to Advanced Cluster Management
+     When user USER1 views Advanced Cluster Management / Overview
+     Then the "Upgrade risk" section displays the current N clusters with high upgrade risk
+      And the "Upgrade risk" section displays the date/time of the last refresh
+      And the "Upgrade risk" section is clickable
+     When user USER1 clicks on the "Upgrade risk" section
+     Then the browser displays the list of the N clusters
+
+
+  Scenario: Displaying the upgrade risk for multiple clusters
```

Comment:
I think this scenario is nice to have eventually, but, if we can cover scenario1 and scenario3, then I think this is not absolutely essential. In other words we give them a sense of the risk via the Overview, and the individual risk at the cluster details level. So when selecting multiple clusters they should already have a 'gauge' of overall risk.

Reply:
@berenss The 'gauge' of overall risk is what I described as:

```
And the "Upgrade clusters" window displays the upgrade risk for each N cluster
```

I would keep the first part of the scenario, ok? It can be implemented as gauge icon, text, or anything else.

Are you maybe proposing to remove this

```
 When user USER1 clicks on the "Upgrade risk" for a cluster
 Then the browser opens the details page on console.redhat.com (Advisor / Cluster / Upgrade risk)
 When user USER1 clicks "Re-evaluate risk"
```
or this

```
 Then a new request is sent to console.redhat.com to get the updated values for the upgrade risks 
 And the "Upgrade clusters" window displays the updated values
 And the "Upgrade clusters" window displays the current date/time as the last refresh date/time
```
or both?

Reply:
Can you please number the scenarios?
I am talking about `Scenario: Displaying the upgrade risk for multiple clusters`
I do not think this is necessary, since we have the scenario1 Overview and the scenario3 display in 1 cluster

Reply:
Ok, removed